src/libpgmoneta/wal.c

```diff
@@ -285,6 +286,12 @@ pgmoneta_wal(int srv, char** argv)
                            pgmoneta_log_error("Could not create or open WAL segment file at %s", d);
                            goto error;
                         }
+
+                        current_wal_filename = pgmoneta_append(current_wal_filename, filename);
+                        current_wal_filename = pgmoneta_append(current_wal_filename, ".partial");
+                        strcpy(config->servers[srv].current_wal_filename, current_wal_filename);
```

Comment:
Is it better if you just use `snprintf` to set the file name directly to `servers[srv].current_wal_filename` ? Other than that, this works nicely.

Reply:
Yes that seems better. Thank you!